ui/webui/resources/css/text_defaults_md.css

```diff
@@ -19,7 +19,7 @@ html {
 }
 
 body {
-  font-family: Poppins, $i18nRaw{fontfamily};
+  font-family: $i18nRaw{fontfamily};
```

Comment:
Do we need to specify `system-ui` here, or is that the default?

Reply:
```
fontfamily: "system-ui, sans-serif",
fontfamilyMd: "system-ui, sans-serif"
```

Reply:
I **think** that's the default when you don't specify it